src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1235,6 +1235,9 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Make_Async_Sub" xml:space="preserve">
     <value>Make Async Sub</value>
   </data>
+  <data name="Multiple_Types" xml:space="preserve">
+    <value>&lt;Multiple Types&gt;</value>
+  </data>
```

Comment:
Does this need to be in both src/Features/Core and src/Features/VisualBasic? I think we regularly reach into the language-independent side since I think that should have internals access.

Reply:
It's actually a bit weird. FeatureResources.resx contains a string for Multiple Types, but FeatureResources.Designer.cs doesn't have it. It looks like I can just keep the VB resource.

Reply:
Yup. Removing the resource from FeaturesResources.resx is the right thing here. This text is only used in VB when hovering over variable declarations with multiple declarators where the type is different (e.g. `Dim x As Integer, y As String`).